OpenAPIService/PowershellFormatter.cs

```diff
@@ -33,6 +33,12 @@ public PowershellFormatter(bool singularizeOperationIds)
             _singularizeOperationIds = singularizeOperationIds;
         }
 
+        static PowershellFormatter()
+        {
+            Vocabularies.Default.AddSingular("(drive)s$", "$1"); // drives does not properly singularize to drive.               
+            Vocabularies.Default.AddSingular("(data)$", "$1"); // exclude the following from singularization.
+        }
```

Comment:
Can we also add exclusions to the singularization rule for the following words:
```suggestion
            Vocabularies.Default.AddSingular("(delta)$", "$1");
            Vocabularies.Default.AddSingular("(quota)$", "$1");
            Vocabularies.Default.AddSingular("(statistics)$", "$1");
        }
```